### PR TITLE
updating wiki with coding guidelines and updating contributing docs

### PR DESCRIPTION
**Pull request by gramhagen**
### Description
Pushed coding guideline info that was previously in contributing.md to wiki: https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines

Updated CONTRIBUTING.md with steps to contribute and reference to wiki as well as putting the code of conduct information that was in previously.

### Motivation and Context
Addresses: #396 

### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING).
- [ ] I have added tests.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,47 +1,73 @@
...
 
-#### Test Driven Development
+Here are the basic steps to get started with your first contribution. Please reach out with any questions.
+1. Use [open issues](https://github.com/Microsoft/Recommenders/issues) to discuss the proposed changes (create an issue describing changes if necessary to collect feedback).
+2. [Fork the repo](https://help.github.com/articles/fork-a-repo/)
+3. Create a new branch to contain changes for the issue. We suggest prefixing the branch with your username and then a descriptive title: (e.g. gramhagen/update_contributing_docs)
+4. Create a test that replicates the issue
+5. Make code changes
+6. Ensure unit tests pass and code style / formatting is consistent (see [wiki](https://github.com/Microsoft/Recommenders/wiki/Coding-Guidelines#python-and-docstrings-style) for more details)
+7. Create a pull request against staging branch
```
[the remark this answers is not in the document]
Comment (yueguoguo) [reply CONTRIBUTING.md]: +1 on this - probably cross reference the wiki page of merging strategy where things are detailed.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,47 +1,73 @@
...
 Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
-## Recommenders Team Contribution Guidelines
+## Steps to Contributing
 
-#### Test Driven Development
+Here are the basic steps to get started with your first contribution. Please reach out with any questions.
+1. Use [open issues](https://github.com/Microsoft/Recommenders/issues) to discuss the proposed changes (create an issue describing changes if necessary to collect feedback).
```
Comment (yueguoguo) [inline CONTRIBUTING.md]: Maybe worth mentioning that it is good practice to tag the issue with the provided labels, which makes easy for users to retrieve issues of interest.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gramhagen) [reply CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (gramhagen) [reply CONTRIBUTING.md]: 👍

**Comment by gramhagen** [comment]
Fixed wiki homepage to add coding guideline to list of topics too

**Pull request merged**